doc/api/cli.md

```diff
@@ -138,6 +138,10 @@ Force FIPS-compliant crypto on startup. (Cannot be disabled from script code.)
 
 Specify ICU data load path. (overrides `NODE_ICU_DATA`)
 
+### `--preserve-symlinks`
```

Comment:
I would put this below `--zero-fill-buffers` -- no particular reason but these are roughly ordered in the likely hood of people using them.